.codeclimate.yml

```diff
@@ -0,0 +1,17 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - php
+  fixme:
+    enabled: true
+  phpmd:
+    enabled: true
+ratings:
+  paths:
+  - "**.php"
+exclude_paths:
+- docs/*
+- tests/*
```

Comment:
We don't use codeclimate as a part of our regular CI toolchain at the moment

Reply:
Fair enough, it's just a copy-paste from other modules. I don't mind removing it